official-antistasi-community/A3-Antistasi
# Additional template test code for manual sanity checks

**!!! Merge after #3120 !!**

## What type of PR is this.
1. [ ] Bug
2. [ ] Change
3. [x] Enhancement

### What have you changed and why?
Information:
Adds ability so spawn units of current selected template as well as fitting vehicles

### Please specify which Issue this PR Resolves.

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 

1. Load into Antistasi
2. Start game with selected Factions
3. execute locally while in large area without Obstacles (Salt sea on Altis) and check if vehicle Spawner spawns every template vehicle with

`[] call A3A_fnc_debugSpawnVics;`

for the unit-spawner use: `[] call A3A_fnc_spawnSelectedTemplateAI;`


********************************************************
Notes:

These Script are only build for debuging
